setup.py

```diff
@@ -61,17 +61,17 @@
     'Programming Language :: Python :: 2',
     'Programming Language :: Python :: 2.7',
     'Programming Language :: Python :: 3',
-    'Programming Language :: Python :: 3.4',
```

Comment:
Are we fine with dropping support for 3.4? I don't have a strong opinion. However, `subprocess.run` is only a [tiny wrapper](https://github.com/python/cpython/blob/3.5/Lib/subprocess.py#L378-L399) around `subprocess.Popen`, so not sure if we really need it.

Also note that in order to duplicate streams (#160), we probably have to use `Popen` anyway, because AFAICS `run` only returns after the process has finished (blocking), but for stream duplication, we need to perform some read/write while the process is running.